### PR TITLE
MC-18516 Delete commitments addition

### PR DESCRIPTION
**Pull request by AlexFerence**
#### Changes made

- Added docs for deleting a commitment

#### Related PRs

- [UI PR](https://github.com/cloudops/cloudmc-ui/pull/2536)
- [Core PR](https://github.com/cloudops/cloudmc-core/pull/2271)

### CODE REVIEW

File: source/includes/administration/_commitments.md [hunk 1]
```diff
@@ -404,4 +404,22 @@ Object: **Product**
...
+
+
+<!-------------------- DELETE COMMITMENT -------------------->
+### Delete commitment
+
+`DELETE /commitments/:id`
+
+Deletes a commitment that has already been created for an organization.
+
+Can only be deleted if a commitment has no closed billing cycles associated to it. This means that a commitment will likely only be able to be deleted within a month of creation at most.
```
Comment (razine-bensari) [inline source/includes/administration/_commitments.md]:
```suggestion
A commitment can only be deleted if it has no closed billing cycles associated to it. This means that a commitment will likely only be able to be deleted within a month of creation at most.
```

File: source/includes/administration/_commitments.md [hunk 2]
```diff
@@ -404,4 +404,22 @@ Object: **Product**
...
+
```
Comment (razine-bensari) [inline source/includes/administration/_commitments.md:426]: We are missing the shell portion of the docs ?

```shell
# Delete organization commitment
curl "https://cloudmc_endpoint/api/v2/commitments" \
   -H "MC-Api-Key: your_api_key"
```

Confirm with me I may be wrong

**Review by razine-bensari — CHANGES_REQUESTED** [review]

File: source/includes/administration/_commitments.md (hunk 2 above)
Comment (AlexFerence) [reply source/includes/administration/_commitments.md:426]: Not there but it seems to be missing in the other operations as well which is why I was suspicious.

Comment (razine-bensari) [reply source/includes/administration/_commitments.md:426]: I think we should add it for consistency, because most other endpoints have it no ?

Comment (AlexFerence) [reply source/includes/administration/_commitments.md:426]: Yup large majority do, will follow up with the commitments one missing

Comment (AlexFerence) [reply source/includes/administration/_commitments.md:426]: https://github.com/cloudops/cloudmc-api-docs/pull/568/commits/fce53d83c298ed74f139e832f3666aa988a91a12 shell script added in this commit

**Review by razine-bensari — APPROVED** [review]
Thank you boss!

**Pull request merged**